sql/sql_class.h

```diff
@@ -1624,10 +1627,10 @@ class Bloom_filter {
   {
     if (!bit_set)
     {
-      void *bit_set_place= alloc_root(mem_root, sizeof(Bit_set));
+      void *bit_set_place= my_malloc(PSI_INSTRUMENT_ME, sizeof(Bit_set), 0);
```

Comment:
No need to have `mem_root` as an input parameter.

Reply:
We can actually keep the mem_root, I retested it again. I think I forgot a make install before: the memroot itself isn't the issue, the recreateing in clear is.